readme.md

```diff
@@ -56,26 +56,28 @@ Note: You're not restricted to digital ocean. Any machine with docker should be
 
        $ git clone https://github.com/speckleworks/SpeckleServer.git
        $ cd SpeckleServer
-       $ npm install
+       $ sudo npm install
 
-5) Follow the instructions in `.env-base` file to configure your server.
+5) Follow the instructions in `.env-base` file to configure your server. Use `sudo nano .env` to edit.
     
 6) Start mongo (create a folder somewhere to store the db): 
 
-       $ mongodb --dbpath /path/to/some/folder
+       $ sudo mongodb --dbpath /path/to/some/folder
     
-7) Start redis in another terminal: 
+7) Start redis in another terminal. You might need to restart server instance after this step. 
 
        $ redis-server
  
 8) Check that both mongo and redis are running OK and that you can connect to them with these two clients:
 
-       $ mongo
-       $ redis-cli
+       $ service mongodb status
+       $ redis-cli ping
     
 9) Start Speckle in a third terminal: 
 
-       $ node server.js
+       $ sudo node server.js
```

Comment:
again, that sudo is risky! would also add instructions or a pointer towards either pm2 [0] or forever [1] as a way to keep the server alive throughout restarts/errors.
[0] http://pm2.keymetrics.io/
[1] https://www.npmjs.com/package/forever